packages/tooltip/src/tooltipUtils.tsx

```diff
@@ -0,0 +1,105 @@
+import { css } from '@leafygreen-ui/emotion';
+
+export function trianglePosition(
+  alignment: string,
+  justification: string,
+  triggerRect: DOMRect | ClientRect | null,
+) {
+  if (!alignment || !justification || !triggerRect) {
+    return '';
+  }
+
+  const borderOffset = 'calc(100% - 1px)';
+
+  const containerSize = 15;
+  const notchSize = 8;
+  const containerOffsetX = (triggerRect.width - containerSize) / 2;
+  const containerOffsetY = (triggerRect.height - containerSize) / 2;
+  const notchOverlap = -notchSize / 2;
+  const containerStyleObj: any = {};
+  const notchStyleObj: any = {};
```

Comment:
We should type these more accurately: 

Something like this I think:
```
const notchStyleObj: {K: string} = {};
```

Reply:
👍